src/components/Icon/BankIcons.js

```diff
@@ -0,0 +1,115 @@
+import NavyFederalCreditUnion from '../../../assets/images/bankicons/navy-federal-credit-union.svg';
+import PNC from '../../../assets/images/bankicons/pnc.svg';
+import RegionsBank from '../../../assets/images/bankicons/regions-bank.svg';
+import SunTrust from '../../../assets/images/bankicons/suntrust.svg';
+import TdBank from '../../../assets/images/bankicons/td-bank.svg';
+import USBank from '../../../assets/images/bankicons/us-bank.svg';
+import USAA from '../../../assets/images/bankicons/usaa.svg';
+
+/**
+ * Returns matching asset icon for bankName
+ * @param {String} bankName
+ * @returns
+ */
+
+function getAssetIcon(bankName) {
+    if (bankName.includes('americanexpress')) {
+        return AmericanExpress;
+    }
+
+    if (bankName.includes('bank of america') || bankName.includes('bankofamerica')) {
+        return BankOfAmerica;
+    }
+
+    if (bankName.startsWith('bbt')) {
+        return BB_T;
+    }
+
+    if (bankName.startsWith('capital one') || bankName.includes('capitalone')) {
+        return CapitalOne;
+    }
+
+    if (bankName.startsWith('chase') || bankName.includes('chase')) {
+        return Chase;
+    }
+
+    if (bankName.includes('charles schwab') || bankName.includes('charlesschwab')) {
+        return CharlesSchwab;
+    }
+
+    if (bankName.startsWith('citibank') || bankName.includes('citibank')) {
+        return CitiBank;
+    }
+
+    if (bankName.startsWith('citizens bank') || bankName.includes('citizensbank')) {
+        return CitizensBank;
+    }
+
+    if (bankName.startsWith('discover ') || bankName.includes('discover.') || bankName === 'discover') {
+        return Discover;
+    }
+
+    if (bankName.startsWith('fidelity')) {
+        return Fidelity;
+    }
+
+    if (bankName.startsWith('huntington bank') || bankName.includes('huntingtonnational') || bankName.includes('huntington national')) {
+        return HuntingtonBank;
+    }
+
+    if (bankName.startsWith('navy federal credit union') || bankName.includes('navy federal credit union')) {
+        return NavyFederalCreditUnion;
+    }
+
+    if (bankName.startsWith('pnc') || bankName.includes('pnc')) {
+        return PNC;
+    }
+
+    if (bankName.startsWith('regions bank') || bankName.includes('regionsbank')) {
+        return RegionsBank;
+    }
+
+    if (bankName.startsWith('suntrust') || bankName.includes('suntrust')) {
+        return SunTrust;
+    }
+
+    if (bankName.startsWith('td bank') || bankName.startsWith('tdbank') || bankName.includes('tdbank')) {
+        return TdBank;
+    }
+
+    if (bankName.startsWith('us bank') || bankName.startsWith('usbank')) {
+        return USBank;
+    }
+
+    if (bankName.includes('usaa')) {
+        return USAA;
+    }
+
+    return DefaultBank;
+}
+
+/**
+* Returns Bank Icon that matches to existing bank icons or default Icon
+* @param {String} bankName
+* @returns
+*/
+
+export default function getBankIcon(bankName) {
+    if (!bankName) {
+        return DefaultBank;
```

Comment:
So can we not get the default credit card icon anymore? We still want that for cards that don't match anything

Reply:
We can do it since we are trying to get Icon for Bank Name by default it should be a Bank Logo. 
I'll make the changes if needed, let me know about the default card icon size whether new size or the old size. Also confirm for default bank size also.

Reply:
Default bank size looks good to me, I think card can be the same.

We definitely want a card icon when there are no matches.

I think adding a param to `getBankIcon` for `isCard` and returning the default card icon would be a good solution for this

Reply:
Updated